inc/tainetworkif.h

```diff
@@ -60,36 +60,23 @@ typedef enum _tai_network_interface_oper_status_t
 /** @brief The transmit alignment status */
 typedef enum _tai_network_interface_tx_align_status_t
 {
-    TAI_NETWORK_INTERFACE_TX_ALIGN_LOSS             = 0x01,
-    TAI_NETWORK_INTERFACE_TX_ALIGN_OUT              = 0x02,
-    TAI_NETWORK_INTERFACE_TX_ALIGN_CMU_LOCK         = 0x04,
-    TAI_NETWORK_INTERFACE_TX_ALIGN_REF_CLOCK        = 0x08,
-    TAI_NETWORK_INTERFACE_TX_ALIGN_TIMING           = 0x10
+    TAI_NETWORK_INTERFACE_TX_ALIGN_LOSS,
+    TAI_NETWORK_INTERFACE_TX_ALIGN_OUT,
+    TAI_NETWORK_INTERFACE_TX_ALIGN_CMU_LOCK,
+    TAI_NETWORK_INTERFACE_TX_ALIGN_REF_CLOCK,
+    TAI_NETWORK_INTERFACE_TX_ALIGN_TIMING
 } tai_network_interface_tx_align_status_t;
```

Comment:
The reason these were defined as bitmaps was that there could be multiple reasons for the transmitter to be out of alignment. So TX_ALIGN_OUT and TX_ALIGN_REF_CLOCK could both be set. A bitmap seemed like a reasonable way to represent this. Is there any reason why you changed this to an enum with incrementing values and modified the attribute to be a list?

Reply:
https://github.com/opencomputeproject/SAI/blob/master/inc/saiport.h#L275-L281

I'm following SAI. This also makes implementing meta library simple.